tests/inventory

```diff
@@ -1,26 +1,199 @@
+#### ALL HOSTS AND CONTAINERS START HERE
+
 [all]
-localhost physical_host=localhost
+localhost ansible_become=True ansible_connection=local ansible_user=root
+infra1
```

Comment:
no just 1 infra, the rpc-o integrated gate can test this with >1 node. That said, the next act will be to break down the actual testing being done and create a matrix of tests that can be run individually.